docs/ERROR_HANDLING.md

```diff
@@ -0,0 +1,70 @@
+# Stark Error Handling
+
+Stark provides its own error handler, StarkErrorHandler, that overrides Angular's ErrorHandler. 
+This handler will dispatch an error which you can treat as you want. 
```

Comment:
"... will dispatch an StarkUnhandledError action to the application Store which you can treat as you want."